assets/css/main.scss

```diff
@@ -1,8 +1,139 @@
 ---
 ---
 
-
-// U.S. Web Design System setup 
+// U.S. Web Design System setup
 $font-path: '../uswds/fonts';
 $image-path: '../uswds/img';
+// U.S. Web Design System variable setup
+@import 'uswds/lib/bourbon';
+@import 'uswds/lib/neat';
+@import 'uswds/lib/normalize';
+@import 'uswds/core/variables';
+
+// Setting typography variables
+{% if site.data.fonts %}
+  // Typography
+  {% if site.data.fonts.sans %}$font-sans: {{ site.data.fonts.sans }};{% endif %}
+  {% if site.data.fonts.serif %}$font-serif: {{ site.data.fonts.serif }};{% endif %}
+  {% if site.data.fonts.normal %}$font-normal: {{ site.data.fonts.normal }};{% endif %}
+  {% if site.data.fonts.bold %}$font-bold: {{ site.data.fonts.bold }};{% endif %}
+{% endif %}
+
+// Setting color variables
+{% if site.data.colors %}
+  // Colors
+
+  // Header colors
+  {% if site.data.colors.header %}
+    $header-bg: {{ site.data.colors.header.header-bg | default: "$color-white" }};
+    $header-title: {{ site.data.colors.header.header-title }};
+    $header-link: {{ site.data.colors.header.header-link }};
+    $header-link-hover: {{ site.data.colors.header.header-link-hover }};
+  {% endif %}
+
+  // Color - Primary
+  {% if site.data.colors.primary %}
+    $color-primary: {{ site.data.colors.primary }};
+  {% endif %}
+  {% if site.data.colors.primary-darker %}
+    $color-primary-darker: {{ site.data.colors.primary-darker }};
+  {% endif %}
+
+  // Color - Text and links
+  {% if site.data.colors.base %}
+    $color-base: {{ site.data.colors.base }};
+    $color-visited: {{ site.data.colors.visited }};
+  {% endif %}
+
+  // Color - Additional variables
+  {% if site.data.colors.color-gray-lightest %}
+    $color-gray-lightest: {{ site.data.colors.color-gray-lightest }};
+  {% endif %}
+
+  // Color - Navigation
+  // Top nav
+  {% if site.data.colors.top-navigation.top-nav-bg %}
```

Comment:
To keep this consistent with some of the other conditional statements this could probably just be `{% if site.data.colors.top-navigation %}` to check if any values exist under `top-navigation`.